lib/lure/repo.go

```diff
@@ -183,3 +190,132 @@ func Execute(pwd string, command string, params ...string) (string, error) {
 
 	return out, nil
 }
+
+func cleanupUpdateBranches(auth Authentication, project Project, repo Repo) error {
+
+	if project.Vcs != Hg {
+		return nil
+	}
+
+	trashBranch := project.TrashBranch
+	if trashBranch == "" {
+		log.Printf("Info: Project has no trash branch defined. Skipping cleanup.")
+		return nil
+	}
+
+	log.Printf("Info: Cleaning up lure branches with no associated PRs.")
+
+	out, err := repo.Cmd("branches", "--active", "--template", "{branches}\n")
+	if err != nil {
+		return err
+	}
+
+	branches := strings.Split(out, "\n")
+	branchPrefix := project.BranchPrefix
+	if branchPrefix == "" {
+		branchPrefix = "lure-"
```

Comment:
We don't use that value anywhere else other then the README. I don't think we can assume that "lure -" will be a default prefix.

Reply:
https://github.com/coveo/lure/blob/3a42b54ded72aa248bfd7e8de8e7e34771b6ed1b/lib/lure/repo.go#L121

Reply:
Thank you I missed that one.

I then think we should use the same variable both places.